tools/caffe_translator/src/main/java/io/mxnet/caffetranslator/Converter.java

```diff
@@ -157,19 +157,24 @@ public String generateMXNetCode() {
             // If the translator cannot translate this layer to an MXNet layer,
             // use CaffeOp or CaffeLoss instead.
             if (generator == null) {
-                if (layer.getType().toLowerCase().endsWith("loss")) {
+                if (layer.getType().toLowerCase().endsWith("loss") && !(layer.getType().equalsIgnoreCase("Accuracy"))) {
```

Comment:
nitpik - if we swap the conditional we can get away with && ?

Reply:
Will fix this code.